website/src/pages/docs/plugins/operation-field-permissions.mdx

```diff
@@ -0,0 +1,9 @@
+# Operation Field permissions
+
+TBD
```

Comment:
can we excluded this page in the meantime?

Reply:
Done :)